README.md

```diff
@@ -22,9 +22,21 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+#### Application Bundle Builders
 
-- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
+Application bundle builders can create a [bundle](bundle.md) directory that includes all of the files and some low-level details for launching an application as a container.
+The application can in-turn rely on certain OS configuration being available when it is launched inside of a container by a runtime.
+The OS configuration will include things such as environment variables, a working directory, mount points, and operating system permissions.
+
+#### Hook Writers
+
+[Hook](runtime-config.md#hooks) authors can extend the functionality of an OCI-compliant runtime by hooking into a containers lifecycle with an external application.
+A hook can manipulate the container using state information passed to the hook at the time of the lifecycle event.
```

Comment:
I don't see a need for this level of detail since we already link to the relevant spec section.  The earlier “extend” sentence and the following “use cases include” sentence are all we need to cover here.